js-src/src/app/map.component.js

```diff
@@ -125,21 +123,23 @@ let MapComponent = Component({
                                 isOpen: 'true',
                                 details: currentCity.name
                             });
-
-                            if (self.currentRound <= self.markers.length) {
-                                self.startCountdown(1);//Initialize Break timer
-                            } else {
-                                //Game ended, navigate to highscore page
-                                this.gameEnded = true;
-                                self.router.navigate(['/highscores', self.game.id, self.player.id])
-                            }
+							
                             self.guessService.getAllGuesses(self.game.id).subscribe(function (guesses) {
                                 for (var i = 0; i < guesses.length; i++) {
                                     if ((guesses[i].player.id != self.player.id) && (guesses[i].city.id == currentCity.id)) {
                                         var pin2 = self.getPinImage(self, guesses[i].player.id);
                                         self.playerMarkers.push({lat: guesses[i].lat, lng: guesses[i].long, img: pin2});
                                     }
                                 }
+								if (self.currentRound < 2) {//Here we have got 3 rounds each game
+									self.startCountdown(1);//Initialize Break timer
+								} else {
+									//Game ended(last round), navigate to highscore page
+									this.gameEnded = true;
+									setTimeout(function () {
+										self.router.navigate(['/highscores', self.game.id, self.player.id]);
+									},2000);
```

Comment:
can we have a button to navigate to the highescores instead of the timer counting down, so the user can review the game as long as he likes? (just an idea, don't do it, if you don;t like it^)